docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
 This page contains instructions for the setup you should go through before you start development work on the deploy system.
 It is intended as a step by step guide and you are encouraged to improve the documentation if any step is not straight forward in its current state.
 
- TODO([#4324](https://github.com/civiform/civiform/issues/4324))This entire page needs to be rewritten. For each item in the list explain what it is and how to install/get it, remove or change the existing documentation below as required. 
```

Comment:
While we're here, might as well fix `Prerequisits` -> `Prerequisites` in the title